### ISSUE TITLE
[Android] Customize the UI

### ISSUE THREAD

**Issue by Emmyalexjnr** [mieszko4/react-native-zoom-us#269]
Please can we have an example code of a customised UI that has most of the Zoom functionality. I just need to be able add a react component to it. The example code in the customised UI part is very skeletal and I have tried using it to achieve this but I can't. Can someone please help me with a customised UI code that has almost all the functionalities of Zoom or one that just has 'number of participants', 'turn on/off audio and video', 'chat' etc. Thank you

**Comment by NJneeraj** [comment]
Yes , It will be helpful If we can get an example of how to set Custom UI  for Zoom in react native

**Comment by NJneeraj** [comment]
Why this is issue has no attention yet?

**Comment by mieszko4** [comment]
I guess because nobody have had time to contribute or maybe not that many people use this feature. Could be because we do not have [iOS part implemented](https://github.com/mieszko4/react-native-zoom-us/issues/113).
